projects/igniteui-angular/src/lib/grids/common/events.ts

```diff
@@ -98,15 +98,19 @@ export interface IColumnSelectionEventArgs extends CancelableEventArgs, IBaseEve
     readonly event?: Event;
 }
 
-export interface ISearchInfo {
+export interface IBaseSearchInfo {
```

Comment:
Only not sure we care to have the base interface public (which this becomes in this file). Other than that, this looks fine

Reply:
Please move the interface to a file that is not exported so that it remains private.